packages/analytics/dashboard-renderer/src/components/DashboardTile.vue

```diff
@@ -61,7 +61,7 @@ const componentData = computed(() => {
       queryReady: props.queryReady,
       chartOptions: props.definition.chart,
       height: props.height - PADDING_SIZE * 2,
-      editable: props.definition.editable,
+      editable: props.context.editable,
```

Comment:
I don't think you need to pass in this prop.

Reply:
done

Reply:
It looks like it's still there...